packages/nimble-components/src/table/index.ts

```diff
@@ -249,6 +252,12 @@ export class Table<
     // the selection checkbox 'checked' value should be ingored.
     // https://github.com/microsoft/fast/issues/5750
     private ignoreSelectionChangeEvents = false;
+    // Map from external slot name to the record ID of the row that should have the slot and
+    // the uniquified slot that the slot should be given.
```

Comment:
```suggestion
    // the uniquified name that the slot should be given.
```

Reply:
Your correction actually isn't correct. Trying to write clear comments with overloaded terms ("slot" in particular) has been difficult, but I'll try to clarify this. What do you think about this?

_Map from the external slot name to the record ID of the row that should have the slot and the unique slot name that the slot should be slotted into._